core/src/fetch_stage.rs

```diff
@@ -190,6 +191,7 @@ impl FetchStage {
                         coalesce,
                         true,
                         in_vote_only_mode.clone(),
+                        false, // unstaked connections
```

Comment:
The convention is that the trailing comment should be the argument name; `is_staked_service` here.
This is kind of "double negative" and confusing. Does "false unstaked" mean "staked"?

Reply:
there's already instances of both "explanation" style comments on bool arguments, as well as "name" style comments.

This is a backport, keep it the same as the original. If we feel strongly enough about the style we should be making modifications to master, not in a backport PR.